react/src/component/PerfTestDashboard.jsx

```diff
@@ -39,21 +39,35 @@ const androidMemoryOptions = [
     { value: 'unknownPss', label: 'unknownPss', color: '#8884d8' },
     { value: 'unknownRss', label: 'unknownRss', color: '#8884d8' }
 ]
+const windowsMemoryOptions = [
+    { value: 'nonpagedSystemMemorySize64', label: 'nonpagedSystemMemorySize64', color: '#007FFF' },
+    { value: 'pagedMemorySize64', label: 'pagedMemorySize64', color: '#FFA500' },
+    { value: 'pagedSystemMemorySize64', label: 'pagedSystemMemorySize64', color: '#8B8970' },
+    { value: 'peakPagedMemorySize64', label: 'peakPagedMemorySize64', color: '#800000' },
+    { value: 'peakWorkingSet64', label: 'peakWorkingSet64', color: '#808000' },
+    { value: 'privateMemorySize64', label: 'privateMemorySize64', color: '#4B0080' },
+    { value: 'workingSet64', label: 'workingSet64', color: '#8884d8' },
+    { value: 'peakVirtualMemorySize64', label: 'peakVirtualMemorySize64', color: '#FFCC00' },
+]
 
 export default class PerfTestDashboard extends React.Component {
     state = {
         perfTestResult: this.props.perfTestResult,
         memoryInfo: undefined,
         batteryInfo: undefined,
```

Comment:
Since we have windows performance char now, could you pls update the `memoryInfo` and `batteryInfo` to `androidMemoryInfo` and `androidBatteryInfo`? Thx

Reply:
Thanks for the review.
I have updated these variables. All the dashboards works well.
![image](https://user-images.githubusercontent.com/10386842/224949696-b4fbaf90-52ff-443c-aa60-8d9ae6637891.png)